src/table/components/head/TableHeadWrapper.tsx

```diff
@@ -80,7 +87,9 @@ function TableHeadWrapper({
               aria-pressed={isCurrentColumnActive}
               onKeyDown={handleKeyDown}
               onMouseDown={() => handleClickToFocusHead(columnIndex, rootElement, setFocusedCellCoord, keyboard)}
-              onClick={(evt: React.MouseEvent) => handleClickToSort(evt, column, changeSortOrder, isInteractionEnabled)}
+              onClick={(evt: React.MouseEvent) =>
+                !menuOpen && handleClickToSort(evt, column, changeSortOrder, isInteractionEnabled)
```

Comment:
Couldn't find a way to test it. menuOpen is false by default and there is no way to set it to true in the test file.

Reply:
So here you are mixing where you put the logic. You extracted `handleClickToSort` in a previous PR to make it testable. Now you check `!menuOpen` before you call the function. I think we can add `!menuOpen` to `isInteractionEnabled`